plugin/evm/customtypes/hashing_test.go

```diff
@@ -43,6 +42,8 @@ import (
 	"github.com/ava-labs/libevm/rlp"
 	"github.com/ava-labs/libevm/trie"
 	"github.com/ava-labs/libevm/triedb"
+
+	mrand "math/rand"
```

Comment:
Is there a reason for this change?

Reply:
Yep, `goimports` or `gci` lint error.

Reply:
You should just not alias it -- there's no reason to.